### DISCUSSION TITLE
This error keeps on occuring

### DISCUSSION THREAD

**Discussion by nonsensibl** [charlesbel/Microsoft-Rewards-Farmer#365] [Q&A]
2023-09-24 12:18:50,760 [ERROR] JavascriptException: Message: javascript error: dashboard is not defined
  (Session info: chrome=117.0.5938.92)
Stacktrace:
        GetHandleVerifier [0x00D3CEB3+45171]
        (No symbol) [0x00CC9101]
        (No symbol) [0x00BBBE1D]
        (No symbol) [0x00BBF9A7]
        (No symbol) [0x00BC0FA9]
        (No symbol) [0x00C1E51B]
        (No symbol) [0x00C0B04C]
        (No symbol) [0x00C1DCFA]
        (No symbol) [0x00C0AE46]
        (No symbol) [0x00BE675E]
        (No symbol) [0x00BE78FD]
        GetHandleVerifier [0x00FF54B9+2897529]
        GetHandleVerifier [0x0103DC6B+3194411]
        GetHandleVerifier [0x01037A41+3169281]
        GetHandleVerifier [0x00DC6130+606960]
        (No symbol) [0x00CD2E7C]
        (No symbol) [0x00CCF008]
        (No symbol) [0x00CCF12F]
        (No symbol) [0x00CC1827]
        BaseThreadInitThunk [0x76C47BA9+25]
        RtlInitializeExceptionChain [0x77CAB79B+107]
        RtlClearBits [0x77CAB71F+191]

**Comment by lourok01** [comment]
do you know now how to do ?

**Comment by VictimaVitae** [comment]
> 2023-09-24 12:18:50,760 [ERROR] JavascriptException: Message: javascript error: dashboard is not defined (Session info: chrome=117.0.5938.92) Stacktrace: GetHandleVerifier [0x00D3CEB3+45171] (No symbol) [0x00CC9101] (No symbol) [0x00BBBE1D] (No symbol) [0x00BBF9A7] (No symbol) [0x00BC0FA9] (No symbol) [0x00C1E51B] (No symbol) [0x00C0B04C] (No symbol) [0x00C1DCFA] (No symbol) [0x00C0AE46] (No symbol) [0x00BE675E] (No symbol) [0x00BE78FD] GetHandleVerifier [0x00FF54B9+2897529] GetHandleVerifier [0x0103DC6B+3194411] GetHandleVerifier [0x01037A41+3169281] GetHandleVerifier [0x00DC6130+606960] (No symbol) [0x00CD2E7C] (No symbol) [0x00CCF008] (No symbol) [0x00CCF12F] (No symbol) [0x00CC1827] BaseThreadInitThunk [0x76C47BA9+25] RtlInitializeExceptionChain [0x77CAB79B+107] RtlClearBits [0x77CAB71F+191]

No idea why people more knowledgeable then me won't help you all, but I think you should reinstall the bot and make sure you have followed all of the [instructions](https://github.com/charlesbel/Microsoft-Rewards-Farmer/discussions/397).
